li-utils/src/main/javaPegasus/com/linkedin/common/urn/UrnUtils.java

```diff
@@ -23,7 +23,7 @@ private UrnUtils() {}
   @Nonnull
   public static DatasetUrn toDatasetUrn(
       @Nonnull String platformName, @Nonnull String datasetName, @Nonnull String origin) {
-    return new DatasetUrn(new DataPlatformUrn(platformName), datasetName, toFabricType(origin));
```

Comment:
`toFabricType` should eventually be deleted imho, but this would be a breaking change for clients